libs/ledger-live-common/src/families/near/specs.ts

```diff
@@ -0,0 +1,132 @@
+import invariant from "invariant";
+import { BigNumber } from "bignumber.js";
+import type { Transaction, NearAccount } from "./types";
+import { getCryptoCurrencyById, parseCurrencyUnit } from "../../currencies";
+import { pickSiblings } from "../../bot/specs";
+import type { AppSpec } from "../../bot/types";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { acceptTransaction } from "./speculos-deviceActions";
+
+const currency = getCryptoCurrencyById("near");
+const minimalAmount = parseCurrencyUnit(currency.units[0], "0.00001");
```

Comment:
this feels very very low value, are you sure it is safe to allow that low value? is it divisible by 2 basically (as the "move 50%" will do 🤔 )

Reply:
It is, NEAR has magnitude of 24 so you could send much, much less than that if you wanted to. This `minimalAmount` is more set in place for staking, to match the functionality that we have in the UI (we don't let users stake, unstake, or withdraw less than this `minimalAmount` to match the NEAR wallet's functionality).

Here's an example of a successful send of `0.0000001 NEAR`: https://explorer.near.org/transactions/DCtUPscPuVdZDxnbZczEeUDfRaHG91Sne4s4XGxhiqmT